analytics/lib/counts.py

```diff
@@ -343,16 +356,32 @@ def do_increment_logging_stat(
     else:
         raise AssertionError("Unsupported CountStat frequency")
 
-    row, created = table._default_manager.get_or_create(
-        property=stat.property,
-        subgroup=subgroup,
-        end_time=end_time,
-        defaults={"value": increment},
-        **id_args,
+    is_subgroup: SQL = SQL("NULL")
+    if subgroup is not None:
+        is_subgroup = SQL("NOT NULL")
+        subgroup = str(subgroup)
+        conflict_args.append("subgroup")
+    extra_column_names = SQL(", ").join(map(Identifier, id_args.keys()))
+    values = SQL(", ").join(map(Literal, id_args.values()))
```

Comment:
These could probably be called `id_column_names` and `id_values`; would be a lot clearer in the SQL below.

Reply:
Done! 👍🏽